web/src/settings_config.ts

```diff
@@ -1002,6 +1002,23 @@ export const stream_privacy_policy_values = {
                 "Must be invited by a subscriber; new subscribers can only see messages sent after they join; hidden from non-administrator users",
         }),
     },
+    support_public: {
+        code: "support-public",
+        name: $t({defaultMessage: "Public, private topics"}),
+        description: $t({
+            defaultMessage:
+                "Organization members can join (guests must be invited by a subscriber); Topics created by others will be hidden from non-administrator users",
+        }),
+    },
+    support_private: {
+        code: "support-private",
+        name: $t({defaultMessage: "Private, private topics"}),
+        description: $t({
+            defaultMessage:
+                "Must be invited by a subscriber; Topics created by others will be hidden from non-administrator users",
+        }),
+    },
+
```

Comment:
This should likely be an independent setting, rather than a private type, in the UI.

I also generally recommend putting "add UI for creating a thing" commits at the end of a branch, after all the commits that make the feature work as designed.

Reply:
Got it.